### ISSUE TITLE
set layer-wise learning rate

### ISSUE THREAD

**Issue by karlTUM** [tflearn/tflearn#179]
Like in caffe, the learning rate can be set as layer-wise by `lr_mult`, in tensorflow I find [tensorflow-worklab](https://github.com/sdemyanov/tensorflow-worklab) has this function. So I would suggest in tflearn there will be layer-wise learning rate option in the future. Thank you.

**Comment by aymericdamien** [comment]
Yes, that would be useful! I will check into that. You can also send a pull request if you already know how to update the code :)
